spec/integration/resource/api600/c7000/storage_pool/create_spec.rb

```diff
@@ -11,7 +11,7 @@
 
 require 'spec_helper'
 
-klass = OneviewSDK::API500::C7000::StoragePool
+klass = OneviewSDK::API600::C7000::StoragePool
 RSpec.describe klass, integration: true, type: CREATE, sequence: seq(klass) do
   include_examples 'StoragePoolCreateExample API500'
```

Comment:
change 500 to 600

Reply:
This is done purposefully. Otherwise I would have to create duplicate api600 folder for storage pool.

Reply:
ok. got it.